mysql/errname.go

```diff
@@ -949,6 +949,8 @@ var MySQLErrName = map[uint16]string{
 	ErrInvalidTxn:                 "invalid transaction",
 	ErrEntryTooLarge:              "entry too large, the max entry size is %d, the size of data is %d",
 	ErrNotImplemented:             "not implemented",
+	ErrInfoSchemaExpired:          "Information schema is out of date: schema failed to update in 1 lease",
```

Comment:
"Please make sure TiDB can connect to TiKV nodes"

Reply:
I think the error message starting with the lower case is more idiomatic.

Reply:
Addressed

Reply:
@lonng I agree with you, however, most of the existing error message has upper case prefix. So it's better to keep this style.